docs/maintainers/deploying.md

```diff
@@ -1,7 +1,26 @@
 ---
-title: Deployment and CI/CD Process
+title: Deployment Guide
 ---
 
+# Deploying DEV
+
+Anyone with the ability to merge PRs on GitHub can deploy the application.
+Remember that this is a shared responsibility, so everyone should deploy code
+occasionally.
+
+If the application needs to be deployed when the code is merged, appending
+`[deploy]` to the merge commit's message will trigger a deploy.
+
+We try to avoid deploying when the site is experiencing a log of traffic. When
```

Comment:
Honestly, we should never be afraid to deploy so I think this can be removed. The only time I would halt deploys is if something is broken on the site. 
```suggestion
When
```